src/config.ts

```diff
@@ -81,9 +81,7 @@ export const setConfig = (newConfig: ConfigOpts): Config => {
   const { signer, provider } = newConfig;
```

Comment:
@rosalinekarr could you add some clear text in the documentation for this method to note that it does this and how to remove options?

Reply:
fixed